### ISSUE TITLE
[USB JOYPAD linux] unreliable axis mapping

### ISSUE THREAD

**Issue by lvd2** [samstyle/Xpeccy#142]
In mapping config GUI, hat axis button presses often are not detected, sometimes they're displayed as "Hat xx" (for pressing hat right, for example) and not working.

Tested in ubuntu 22.04 and gentoo, xpeccy commit 3e9e35a07fee4966617a3f7142bbf35f85ac9bce

**Comment by samstyle** [comment]
Переделал на QGamepad. Должно вылечиться. Номера кнопок-осей могут не совпадать с SDL-овскими, и Qt нужен 5.7 и выше. То, что раньше было Hat в SDL (крестовина) теперь тоже считаются кнопками.

**Comment by lvd2** [comment]
чот в a791f02502bb1d42b05a0f4c1429dde80f3664d5 вообще джойпад не видится.
В консоль пишет gamepad: 1318 ""
Но в окошке настроек none

**Comment by samstyle** [comment]
ID у него не 0. Имя почему-то везде пустое. В настройках название я, видимо, забыл поменять. В биндах геймпада он кнопки-оси опознаёт, или тоже тишина? Мой Ligitech F310 работает, а больше проверять не на чем...

**Comment by lvd2** [comment]
Да, не показывает имя джойстика, но всё кнопки назначаются. Теперь другая проблема -- если на джойстике держать кнопку, то игра ощутимо замедляется, при этом звук не рвётся, а растягивается (как будто если прерывания в спектруме стали приходить реже, но аигрек тактируется той же частотой). фпс из ледов продолжает стабильно казать 50.
